index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
- [Optional ] Please it would be better to fix the width of the header section so it matches the design 
your code 
![image](https://user-images.githubusercontent.com/74744787/219194550-f89c0f97-6492-4cf4-922e-fc8ca69c9316.png)
Figma design
![image](https://user-images.githubusercontent.com/74744787/219195409-4f6d2517-0bb9-47eb-871b-92ca1ace0db0.png)